### PR TITLE
Run Lisflood notebook on SRC instead of Cartesius

### PR DESCRIPTION
**Pull request by sverhoeven**
Should be merged after #264

### CODE REVIEW

File: docs/examples/lisflood.ipynb [hunk 1]
```diff
@@ -15,7 +15,7 @@
    "source": [
```
Comment (SarahAlidoost) [inline docs/examples/lisflood.ipynb:15]: <pre class="ql-syntax" spellcheck="false">Line #1.    model_mask = "/mnt/data//climate-data/aux/LISFLOOD/model_mask.nc"
</pre><p>Please remove one of <code>/</code> after <code>data</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/265/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

**Review by SarahAlidoost — COMMENTED** [review]
@sverhoeven thanks. the changed paths look good :+1: . I couldn't checkout this branch on SRC to run the notebook, will try later.

File: docs/examples/lisflood.ipynb (hunk 1 above)
Comment (sverhoeven) [reply docs/examples/lisflood.ipynb:15]: done

**Comment by sverhoeven** [comment]
@SarahAlidoost did you have a chance to run the notebook on SRC?

**Comment by sverhoeven** [comment]
Created https://github.com/eWaterCycle/infra/pull/110 to document install steps needed for testing procedure on SURF Research Cloud

**Review by SarahAlidoost — APPROVED** [review]
@sverhoeven the notebook works like a charm :+1:

**Comment by sverhoeven** [comment]
Great to hear

**Pull request merged**